publish.gradle

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2019 Cyface GmbH
+ *
+ * This file is part of the Cyface SDK for Android.
+ *
+ * The Cyface SDK for Android is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Cyface SDK for Android is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the Cyface SDK for Android. If not, see <http://www.gnu.org/licenses/>.
+ */
+/**
+ * Gradle's publish file for this module.
+ *
+ * @author Armin Schnabel
+ * @version 1.0.0
+ * @since 5.0.0-beta2
+ */
+
+configure(subprojects.findAll({ it.name in ["persistence", "synchronization", "datacapturing"] })) {
+    pr ->
+        apply plugin: 'com.android.library'
+        apply plugin: 'maven-publish'
+
+        publish.dependsOn 'assemble'
+
+        // Load credentials
+        def properties = new Properties()
+        properties.load(new FileInputStream("local.properties"))
```

Comment:
The account and token to access the Github Package Registry (which can currently only be accessed with a Github account, even for open source projects) are in the local.properties file which is not commited of course.